pkg/tests/ossm/smoke_test.go

```diff
@@ -65,7 +65,7 @@ func TestSmoke(t *testing.T) {
 
 			t.LogStepf("Create SMCP %s and verify it becomes ready", fromVersion)
 			assertSMCPDeploysAndIsReady(t, fromVersion)
-			assertRoutesExist(t)
+			// assertRoutesExist(t)
```

Comment:
This will fail for all versions < 2.4? Could you please add a comment explaining why it's commented out?

Reply:
I see, you are installing SMCP which is actually no longer supported just to verify that upgrade works from that version. In that case, please just add a comment explaining that.

Reply:
@FilipB better now